main.go

```diff
@@ -12,15 +12,15 @@ import (
 
 func main() {
 	// Init file reader
-	file, err := datastore.NewDB()
+	file, client, err := datastore.NewDB()
```

Comment:
Order the imports

Reply:
How should I order these inputs? I haven't found too much information about the correct order.

Reply:
```
Go Core

Your project

Thirdparty 
```

Reply:
Done